force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,93 @@
+            <lightning-input
+                type="number"
+                name="userInterface"
+                label="User Interface: 25%"
+                value={userInterfaceScore}
+                onchange={handleChange}
+            ></lightning-input>
+            <lightning-input
+                type="number"
+                name="testDebugDeploy"
+                label="Testing, Debugging, and Deployment: 22%"
+                value={testingScore}
+                onchange={handleChange}
+            ></lightning-input>
+        </div>
+
+        <!-- Display results -->
+
+        <div slot="footer">
+            <h1 class="slds-text-heading_large slds-text-title_bold slds-text-title_caps">Certification Result</h1>
+            <p class="slds-text-heading_medium">Certification Score: <span class="slds-text-color_destructive">{certificationScore}%</span></p>
+            
+            <div>
+                <h1 class="slds-text-heading_large">Previous Attempt</h1>
+                <lightning-tabset> <!-- making it tabular -->
+                    <div for:each={attemptHistory} for:item="attempt" key={attempt.Id}>
+                            <lightning-tab label={attempt.Id}>
+                                <c-score-information
+                                    score={attempt.Score}
+                                    attempt-id={attempt.Id}
+                                    number-of-questions={numberOfQuestions}
+                                    ondeleteattempt={deleteAttemptHandler}
+                                ></c-score-information> <!-- calling child component -->
+                            </lightning-tab>
+                    </div>
+            </lightning-tabset>
+            </div>
+            
+            <template lwc:if={showResources}> <!-- if didn't pass, show resources -->
+                <ul class="slds-text-align_left slds-text-heading_small slds-p-top_medium">
+                    <li class="slds-text-title_bold">
+                        Resources:
+                    </li>
+                    <li>
+                        <a href="https://trailhead.salesforce.com/en/credentials/platformdeveloperi">PD1 Exam Guide</a>
+                    </li>
+                    <li>
+                        <a href="https://focusonforce.com/courses/salesforce-certified-platform-developer-1-exams">Focus On Force</a>
+                    </li>
+                    <li>
+                        <a href="https://courses.salesforcementor.com/p/salesforce-development-apex-fundamentals">Salesforce Mentor</a>
+                    </li>
+                </ul>
+            </template>
+            <template lwc:elseif={showGoodJob}> <!-- if pass, show good job -->
+                <p>You are ready for your first developer job!</p>
+            </template>
+            <template lwc:else> <!-- else, show statement below -->
+                <p>Press calculate to see your score.</p>
+            </template>
```

Comment:
```suggestion
            <template lwc:elseif={showGoodJob}>
                <p>You are ready for your first developer job!</p>
            </template>
            <template lwc:else>
                <p>Press calculate to see your score.</p>
            </template>
```
I would just remove these comments not really necessary beyond being a placeholder as the else if statement is clean enough.